build.gradle

```diff
@@ -307,12 +321,11 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version:'0.1.0'
   compile group: 'com.github.hmcts', name: 'java-logging', version: '5.1.7'
   compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.2.9.RELEASE'
+  // For the below: https://stackoverflow.com/questions/64994034/bootstrap-yml-configuration-not-processed-anymore-with-spring-cloud-2020-0
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-bootstrap'
```

Comment:
This is needed as the upgraded spring versions don't parse the bootstrap.yaml file that we use. Ideally we should remove the file but that requires a number of changes falling out of the scope of a CVE ticket.
This library, provided by spring, re-instates parsing of bootstrap.yaml.